core/static/css/_main.scss

```diff
@@ -441,7 +441,34 @@ article{
     margin-right: 10px;
     float: left;
   }
+  .block-Image{
+    .image{
+      margin: 0 0 10px 0;
+    }
+    .credit{
+      margin-top: 10px;
+      font-size: 0.8em;
+      text-transform: uppercase;
+      color: $grey;
+    }
+    .left-aligned{
+      float: left;
+      margin: 20px 20px 20px 0;
+    }
+    .right-aligned{
+      float: right;
+      margin: 20px 0px 20px 20px;
+    }
+    .full-width{
+      img{
+        margin: 0 auto;
+      }
+      .credit{
+        text-align: center;
+      }
 
```

Comment:
Do we need to add any max-width or height properties? It is set to pull in the original width image which may not be the correct behaviour, especially once we start uploading high res images